api/apiFunctions.test.tsx

```diff
@@ -24,294 +22,370 @@ jest.mock('./apiFunctions', () => {
+          team: '66218f22b40deef340f8',
+          correct: false,
+        },
+        '6628077faeeedd272637': {
+          team: '6621b30ea57bd075e9d3',
+          correct: false,
+        },
+        '66174f2362ec891167be': { team: '6621b30ea57bd075e9d3', correct: true },
+      };
+      const response = { data: weeklyPicks };
 
-    apiFunctions.getAllWeeklyPicks.mockResolvedValue(response);
+      apiFunctions.getAllWeeklyPicks.mockResolvedValue(response);
 
-    const result = await apiFunctions.getAllWeeklyPicks();
+      const result = await apiFunctions.getAllWeeklyPicks();
 
-    expect(result).toEqual(response);
+      expect(result).toEqual(response);
+    });
   });
-});
 
-describe('getCurrentUserEntries()', () => {
-  it('should get all user entries and check if eliminated is equal to true', async () => {
-    const userEntries = [
-      {
-        eliminated: true,
-        league: {},
-        name: 'name',
-        selectedTeams: [],
-        user: '000',
-        $id: '000',
-      },
-    ];
-
-    apiFunctions.getCurrentUserEntries.mockResolvedValue(userEntries);
-
-    const result = await apiFunctions.getCurrentUserEntries();
-
-    for (const entry of result) {
-      expect(entry.eliminated).toEqual(true);
-    }
-  });
+  describe('getCurrentUserEntries()', () => {
+    it('should get all user entries and check if eliminated is equal to true', async () => {
+      const userEntries = [
+        {
+          eliminated: true,
+          league: {},
+          name: 'name',
+          selectedTeams: [],
+          user: '000',
+          $id: '000',
+        },
+      ];
+
+      apiFunctions.getCurrentUserEntries.mockResolvedValue(userEntries);
+
+      const result = await apiFunctions.getCurrentUserEntries();
+
+      for (const entry of result) {
+        expect(entry.eliminated).toEqual(true);
+      }
+    });
 
-  it('should get all user entries and check if eliminated is equal to false', async () => {
-    const userEntries = [
-      {
-        eliminated: false,
-        league: {},
-        name: 'name',
-        selectedTeams: [],
-        user: '000',
-        $id: '000',
-      },
-    ];
-
-    apiFunctions.getCurrentUserEntries.mockResolvedValue(userEntries);
-
-    const result = await apiFunctions.getCurrentUserEntries();
-
-    for (const entry of result) {
-      expect(entry.eliminated).toEqual(false);
-    }
-  });
+    it('should get all user entries and check if eliminated is equal to false', async () => {
+      const userEntries = [
+        {
+          eliminated: false,
+          league: {},
+          name: 'name',
+          selectedTeams: [],
+          user: '000',
+          $id: '000',
+        },
+      ];
+
+      apiFunctions.getCurrentUserEntries.mockResolvedValue(userEntries);
+
+      const result = await apiFunctions.getCurrentUserEntries();
+
+      for (const entry of result) {
+        expect(entry.eliminated).toEqual(false);
+      }
+    });
 
```

Comment:
The intendentation for this test seems to not match the other tests in this file

Reply:
@choir27 looks the same to me.